test/bench/palindrome.mo

```diff
@@ -1,6 +1,6 @@
 // The 2 palindrome implementations from
 // "There and Back Again", by Olivier Danvy and Mayer Goldberg
-//
+//MOC-FLAG --force-gc
```

Comment:
I think the heap usage is too small for two runs have much effect. In #3090 I add another bench test, alloc.mo, that does more allocation to reveal a slight win on the second run (but really very small)

Reply:
Oh crap, I pushed to you branch by mistake. I think I must have fail to switch branch. Will fix.

Reply:
Now fixed with my two commits below.

Reply:
I have killed the additional commits.